traffic_ops/app/lib/API/User.pm

```diff
@@ -226,6 +226,120 @@ sub update {
 
 }
 
+# Create
+sub create {
+	my $self = shift;
+	my $params = $self->req->json;
+	
+	if ( !&is_oper($self) ) {
+		return $self->forbidden();
+	}
+
+	my $name = $params->{username};
+	if ( !defined($name) ) {
+		return $self->alert("Username is required.");
```

Comment:
you check for username, fullName and email in the is_valid method. so you don't have to do that here.